packages/ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -15,6 +15,7 @@
         class="va-dropdown__content-wrapper"
```

Comment:
Demo seems to completely broken.

`/#/demo/vuestic-components/va-dropdown/VaDropdown.demo.vue`